deploy/chart/devfile-registry/templates/deployment.yaml

```diff
@@ -80,32 +74,55 @@ spec:
             scheme: HTTP
           initialDelaySeconds: 15
           periodSeconds: 10
-          timeoutSeconds: 3
-        startupProbe:
-          httpGet:
-            path: /viewer
-            port: 8080
-            scheme: HTTP
-          initialDelaySeconds: 30
-          periodSeconds: 1
-          timeoutSeconds: 10          
+          timeoutSeconds: 3         
         resources:
           limits:
             memory: {{ .Values.devfileIndex.memoryLimit }}
           requests:
             memory: 64Mi
         env:
-          - name: DEVFILE_VIEWER_ROOT
-            value: "/viewer"
           - name: REGISTRY_NAME
             value: {{ .Values.telemetry.registryName }}
           - name: TELEMETRY_KEY
             value: {{ .Values.telemetry.key }}
-        volumeMounts:
-        - name: viewer-config
-          mountPath: "/app/config"
-          readOnly: false
-
+          {{- if .Values.global.headless }}
+          - name: REGISTRY_HEADLESS
+            value: "1"
+          {{- end }}
+      {{- if not .Values.global.headless }}
+      - image: "{{ .Values.registryViewer.image }}:{{ .Values.registryViewer.tag }}"
+        imagePullPolicy: {{ .Values.registryViewer.imagePullPolicy }}
+        name: registry-viewer
+        ports:
+        livenessProbe:
+          httpGet:
+            path: /viewer
+            port: 3000
+            scheme: HTTP
+          initialDelaySeconds: 15
+          periodSeconds: 10
+          timeoutSeconds: 3
+        readinessProbe:
```

Comment:
Can we also add the startupProbe back in? If for whatever reason the viewer's image doesn't match the registry content then it'll revalidate the cache upon startup.